### PR TITLE
Template metadata

### PR DESCRIPTION
**Pull request by hansvancalster**
Start implementing #22

### CODE REVIEW

**Comment by ThierryO** [comment]
The Docker CI fails, because this branch does not contain a Dockerfile. That is solved when your either merge this into the master or rebase the current branch onto the master.

**Comment by florisvdh** [comment]
Will merge master into this branch to do that.

**Comment by hansvancalster** [comment]
Closes #22

File: R/create_protocol.R [hunk 1]
```diff
@@ -15,17 +15,18 @@
...
+#' name
 #' @param date A character string of the date in ISO 8601 format (YYYY-MM-DD)
-#' @param reviewers A character string for reviewers of the form First name
-#' Last name and multiple authors separated by a comma
+#' @param reviewers A character vector for reviewers of the form First name
+#' Last name
 #' @param file_manager A character string for the name of the document
 #' maintainer of the form First name Last name
-#' @param revision A version number of the form `YYYY.##`.
+#' @param version_number A version number of the form `YYYY.##`.
```
Comment (florisvdh) [inline R/create_protocol.R:25]: Excellent :+1:

File: inst/rmarkdown/templates/template_sfp_en/skeleton/skeleton.Rmd [hunk 2]
```diff
@@ -41,71 +48,18 @@ opts_chunk$set(
 
 ```{r metadata}
 metadata <- data.frame(
-  Reviewers = "{{{reviewers}}}",
-  Documentbeheerder = "{{{file_manager}}}",
-  Procedure = "{{{procedure}}}",
-  Revisie = "{{{revision}}}"
+  Reviewers = paste(params$reviewers, collapse = ", "),
+  File_manager = params$file_manager,
+  Procedure = params$procedure,
```
Comment (florisvdh) [inline inst/rmarkdown/templates/template_sfp_en/skeleton/skeleton.Rmd:53]: @hansvancalster is 'procedure' a term needing explanation (with regard to other terms) within Readme?

Comment (hansvancalster) [reply inst/rmarkdown/templates/template_sfp_en/skeleton/skeleton.Rmd:53]: "procedure" is the term we used up till now (docx style protocols), but when I started transition to GitHub I have used protocol-code (in the Readme.md of protocols repo). So they are synonyms, but for consistency I will change all instances of procedure to protocol-code.

Comment (hansvancalster) [reply inst/rmarkdown/templates/template_sfp_en/skeleton/skeleton.Rmd:53]: Done in 4f365d9

Comment (florisvdh) [reply inst/rmarkdown/templates/template_sfp_en/skeleton/skeleton.Rmd:53]: Thanks!

**Pull request merged**